mapper/brokermeta_test.go

```diff
@@ -1,5 +1,6 @@
 package mapper
 
+/*
```

Comment:
remove?

Reply:
fixed - forgot I had to update this test to pass